src/Core/Geographic/Extent.js

```diff
@@ -263,4 +271,33 @@ Extent.prototype.intersect = function intersect(bbox) {
              this.north() <= other.south(this._internalStorageUnit));
 };
 
+Extent.prototype.set = function set(west, east, south, north) {
```

Comment:
I'm a bit reluctant to add code that isn't used at least once in our code base.
(we can merge it anyway if you *know* it'll be useful)